### ISSUE TITLE
start-service keep popup Accessibility Access

### ISSUE THREAD

**Issue by jheroy** [koekeishiya/yabai#1823]
Exec `yabai --start-service`
popup: 

<img width="461" alt="image" src="https://github.com/koekeishiya/yabai/assets/504766/59bf8c57-5cac-4b69-b90e-c8f25d769d17">
open switch:

<img width="475" alt="image" src="https://github.com/koekeishiya/yabai/assets/504766/5c1a0a5e-7dc0-44b0-aaf3-16f17116993c">

then exec `yabai --start-service` again 
<img width="461" alt="image" src="https://github.com/koekeishiya/yabai/assets/504766/12cdd107-acf3-424e-b8bc-9e66b7320ac4">

I tried delete `yabai` item in accessibility, doesn't work.

**Comment by vietvudanh** [comment]
Same problem, and when it shows up, my MacOS (13.4.1) hang, need to restart?

**Comment by koekeishiya** [comment]
You need to codesign the binary when buildinc fron source, as mentioned in the wiki.

**Comment by vietvudanh** [comment]
> You need to codesign the binary when building from source, as mentioned in the wiki.

Thank you, I read the wiki again, looks like that section is not there last time I installed yabai. Now it works.
